tests/app/forms/field_handlers/test_dropdown_handler.py

```diff
@@ -102,3 +104,17 @@ class TestForm(Form):
     assert form.test_field.description == ""
     assert form.test_field.default == ""
     assert form.test_field.choices == expected_choices
+
+
+def test_build_choices_no_dynamic_or_static_choices(
+    value_source_resolver, rule_evaluator
+):
+    handler = DropdownHandler(
+        dynamic_radio_options_no_static_options(),
+        value_source_resolver,
+        rule_evaluator,
+        error_messages,
+    )
+    with pytest.raises(InvalidSchemaConfigurationException) as context:
+        handler.choices  # pylint: disable=pointless-statement
```

Comment:
Just need asserting.
```suggestion
        assert handler.choices
```